packages/jaeger-ui/src/constants/default-config.tsx

```diff
@@ -27,49 +27,49 @@ export default deepFreeze(
         dagMaxNumServices: FALLBACK_DAG_MAX_NUM_SERVICES,
         menuEnabled: true,
       },
-      menu: [
-        {
-          label: 'About Jaeger',
-          items: [
-            {
-              label: 'Website/Docs',
-              url: 'https://www.jaegertracing.io/',
-            },
-            {
-              label: 'Blog',
-              url: 'https://medium.com/jaegertracing/',
-            },
-            {
-              label: 'Twitter',
-              url: 'https://twitter.com/JaegerTracing',
-            },
-            {
-              label: 'Discussion Group',
-              url: 'https://groups.google.com/forum/#!forum/jaeger-tracing',
-            },
-            {
-              label: 'Online Chat',
-              url: 'https://cloud-native.slack.com/archives/CGG7NFUJ3',
-            },
-            {
-              label: 'GitHub',
-              url: 'https://github.com/jaegertracing/',
-            },
-            {
-              label: `Jaeger ${getVersion().gitVersion}`,
-            },
-            {
-              label: `Commit ${getVersion().gitCommit.substring(0, 7)}`,
-            },
-            {
-              label: `Build ${getVersion().buildDate}`,
-            },
-            {
-              label: `Jaeger UI v${version}`,
-            },
-          ],
-        },
-      ],
+       menu: [
```

Comment:
why this formatting change?

Reply:
Somehow my IDE probably messed up. I will restore the old formatting!